OJP/OJP_Trips.xsd

```diff
@@ -106,18 +107,54 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
-	<xs:group name="TripPolicyGroup">
+	<xs:complexType name="AccessModesListOfEnumerations">
```

Comment:
If that structure isn't used for trips any more, it could be moved to OJP_PlaceSupport.xsd

Reply:
done